cel/options.go

```diff
@@ -29,6 +29,20 @@ import (
 	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
 )
 
+// These constants beginning with FEATURE_ enable optional behavior in
+// the library.  See the documentation for each constant to see its
+// effects, compatibility restrictions, and standard conformance.
+const (
+	_ = iota
+
+	// Disallow heterogeneous aggregate (list, map) literals.
+	// Note, it is still possible to have heterogeneous aggregates when
+	// provided as variables to the expression, as well as via conversion
+	// of well-known dynamic types, or with unchecked expressions.
+	// Affects checking.  Provides a subset of standard behavior.
+	FEATURE_DISABLE_DYNAMIC_AGGREGATE_LITERALS
```

Comment:
This was counter-intuitive to me too, but the convention within the Go standard libraries for exported constants is `CamelCase`, and I believe that's the convention used elsewhere within CEL.

Reply:
Done.